tensorflow/contrib/gan/python/losses/python/losses_impl.py

```diff
@@ -384,7 +381,7 @@ def wasserstein_gradient_penalty(
     # root. Note tf.norm does not add epsilon.
     slopes = math_ops.sqrt(gradient_squares + epsilon)
     penalties = slopes / target - 1.0
-    if mode == 'one_sided':
+    if one_sided is True:
```

Comment:
`if one_sided is True` -> `if one_sided`?

Reply:
Will fix together with lint errors introduced in the last commit. Sorry about that ;)